### ISSUE TITLE
windows - Spurious timing test failures on windows.

### ISSUE THREAD

**Issue by garyfurnish** [boostorg/thread#133]
Related to [Trac 9856](https://svn.boost.org/trac10/ticket/9856) in test_9856

> The fact that Thread doesn't return timeout up to 10% of the time on Windows is wontfix.

Can we at least change the threshold for the test on windows so that we don't get spurious failures?  

This was tested in 3222938.

**Comment by viboes** [comment]
Could you tell me on which platform and compiler you got the issue? are those run in http://www.boost.org/development/tests/develop/developer/thread.html?

**Comment by garyfurnish** [comment]
This is on Windows 10 insiders with Visual Studio 17.4 Preview.  I think it was also failing on 17.3 on windows 10, but I can't guarantee that.  I could rerun with regular 17.3.  This is a 4/8 sandy bridge machine.

**Comment by viboes** [comment]
Could you tell me which % threashold is good for Windows?

**Comment by garyfurnish** [comment]
If I run it alone the highest I've gotten was 13.8% and 15% might be a reasonable target.
If I use b2 -j8 (num hyperthreads) to run the tests then it can go at least 38.2%, maybe higher.  I suspect this test will always fail if run in parallel with other tests.  Run alone its probably reasonable.

**Comment by viboes** [comment]
I guess that the testers are using -jx

I will set the threshold to 40% :(

**Comment by viboes** [comment]
https://github.com/boostorg/thread/commit/3ee73c866de658a8e697ce74e62b05824fa93b5e

**Comment by garyfurnish** [comment]
Tests clean under load, closing as fixed.
